pcl_conversions/CMakeLists.txt

```diff
@@ -1,11 +1,21 @@
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 3.7)
```

Comment:
I picked 3.7 because its the oldest version of CMake that Melodic is targeting (Debian Stretch).  I wouldn't have any objections to lowering it to 3.6.3, but there's probably not much benefit since this is a melodic-devel branch anyway.

Reply:
It restricts anyone building from source on a system that has 3.6.3 for no real justification.

If we want to match the lowest in Melodic it would be 3.7.2 that's why I was wondering where 3.7 came from as it was in between, so allowing users having 3.7.0 to compile this package but not people have 3.6.3.